web/src/components/block-content/embedded-component.js

```diff
@@ -28,6 +30,10 @@ function EmbeddedComponent(component) {
       return <CovidGraphPanel />;
     case "HarvardGraph":
       return <HarvardGraph />;
+    case "HarvardDailyGraph":
```

Comment:
I realized this switch statement is truly not sustainable... this is OK for now but let's think of some alternatives for this as we grow our custom visualizations

Reply:
I'm struggling to think of what could work given the restraints of Sanity.  I'm sure we want to keep the number of schemas in Sanity low so handling the different components would most likely have to happen on the front end so that leads back to the embedded component solution which I agree isn't sustainable.

Perhaps adding fields to embedded component like `category` and `component`. In this case it could be like
```bash
├───COVID
│   ├───CovidMap
│   ├───CountryGraph
│   ├───StateGraphCumulative
│   ├───StateGraphDaily
│   ├───CovidGraphPanel
│   ├───HarvardCumulativeGraph
│   ├───HarvardDailyGraph
│   └───HarvardGraphPanel
└───OTHER
    └───WheelOfFortune
```
where `embedded-component.js` imports some `COVID` component from the covid-dashboard directory which has the switch case stuff for all the covid related interactives.

Reply:
IDK if that really solves the issue though. Just adds another layer of abstraction.